src/Ilios/CoreBundle/Tests/DataLoader/UserData.php

```diff
@@ -70,7 +68,7 @@ protected function getData()
             'cohorts' => ['1'],
             'reminders' => ['1', '2'],
             'pendingUserUpdates' => [],
-            'permissions' => ['2', '3', '1']
+            'permissions' => ['1', '2', '3']
```

Comment:
Yup, linux and OSX process this differently.  No idea why.  Maybe just to annoy me.